config/the-config_top-section.md

```diff
@@ -371,19 +364,19 @@ Saves memory by disabling old-fashioned styles. With this define, these styles w
 ```
 
 ### ENABLE_ALL_EDIT_OPTIONS
-The following "umbrella" define is equivalent to separately defining:  
-DYNAMIC_BLADE_LENGTH  
-DYNAMIC_BLADE_DIMMING  
-DYNAMIC_CLASH_THRESHOLD  
-SAVE_VOLUME  
-SAVE_BLADE_DIMMING  
-SAVE_CLASH_THRESHOLD  
-SAVE_COLOR_CHANGE  
-*Note - `#define SAVE_STATE` is different, as that encompasses:  
-SAVE_VOLUME  
-SAVE_PRESET  
-SAVE_COLOR_CHANGE  
-SAVE_DYNAMIC_DIMMING
+The following "umbrella" define is equivalent to separately defining:
+DYNAMIC_BLADE_LENGTH
+DYNAMIC_BLADE_DIMMING
+DYNAMIC_CLASH_THRESHOLD
+SAVE_VOLUME
+SAVE_BLADE_DIMMING
+SAVE_CLASH_THRESHOLD
+SAVE_COLOR_CHANGE
+*Note - `#define SAVE_STATE` is different, as that encompasses:
+SAVE_VOLUME
+SAVE_PRESET
+SAVE_COLOR_CHANGE
+SAVE_BLADE_DIMMING (From ProffieOS 7.8 onward)
```

Comment:
This hasn't actually been ported to the 7.x branch yet.
Right now it's only on the main branch, which will eventually become 8.x
Maybe it should though...

Reply:
Ahh. Yes. I guess it depends on whether the 7.x branch is still open for fixes and therefore patch releases? Then it would be nice to cherry pick this into the 7.x branch and have this go out with a 7.8 release at some point.

If not I can update the text to say 8.x, or this can be left as is and the PR kept open until this fix gets into a future release?

Reply:
I think you're right about this code going into 7.8.
We can just leave this PR open until that happens.

Reply:
This should be safe to merge now with the 7.8 release? I think I saw the commit for this change in there? :)

Reply:
Sorry, I forgot about this when I made the 7.8 zip.

Reply:
Bumped this to say 7.9